BuildaGitServer/Base/Authentication.swift

```diff
@@ -69,4 +73,16 @@ extension ProjectAuthenticator: KeychainStringSerializable {
             self.secret
             ].joinWithSeparator(":")
     }
+
+    internal static func createEnterpriseService(host: String) -> GitService? {
+        guard let url = NSURL.init(string: "http://\(host)") else { return nil }
+        do {
+            let response = try NSString.init(contentsOfURL: url, encoding: NSASCIIStringEncoding)
+            if response.lowercaseString.containsString("github") {
```

Comment:
You're right, that's my worry. I wonder if there isn't an API call appropriate for this, maybe even trying to list all repositories on that server with `/repos` or something like that. Again, I'm not of much use because I don't have an Enterprise server at my disposal :/